src/views/partials/labels.handlebars

```diff
@@ -1,22 +1,23 @@
-<div>
-    <h3>Labels</h3>
-    <input type="checkbox" id="edit_labels">
-    <label for="edit_labels">Close</label>
+<div class="edit-label-container">
+        <h3 class="edit-labels-title">Labels:</h3>
+        <input type="button" class="close-edit-labels" id="close-edit-labels" value="X">
 
     <form action="/board/card/labels/{{boardId}}/{{cardId}}" method="POST" id="editLabels">
         {{!-- List Labels --}}
-        <div id="list-of-labels">
+        <div class="list-of-labels" id="list-of-labels">
             {{#each labels}}
                 <div class="label-input-container">
                     <input type="checkbox" id="card_label_{{@index}}" name="labels[]" value="{{this._id}}" checked>
-                    <label for="card_label_{{@index}}"><p class="card-label-highlight" style="background-color: {{this.color}};"></p><p>{{this.text}}</p></label>
+                    <label for="card_label_{{@index}}"><p class="card-label-highlight" style="background-color: {{this.color}};"></p>{{this.text}}</label>
```

Comment:
because you changed this, some of the jquery needs to be changed in the board-page relating to making labels, but I can handle changing that.